docs/primer_short.md

```diff
@@ -0,0 +1,146 @@
+# Test-negative design: short primer
+
+## Draft outline
+
+- What is VE
+  - Known biases with any VE estimator
+- What is a TND study
+- There are 8 kinds of people:
+  - Vaccinated vs. not
+  - Infected with the focal pathogen vs. not
+  - Seek a test vs. not
+  - Test positive vs. not
```

Comment:
re: vaccination status. This is an interesting point; I'll break it off into another issue, to see what this does.

If only unvaccinated people claimed to be vaccinated, then this would down-bias estimated VE in a pretty straightforward way. But if vaccinated people forget or don't have their records checked correctly (which seems the more likely thing), then it seems like this should also down-bias VE? I'll check on that.

re: 16. good catch

Reply:
cf #6